### PR TITLE
adding-server-reviews

### PR DESCRIPTION
**Pull request by nadaa**
adding reviews in server side for admin login

### CODE REVIEW

**Comment by abdulazeeznaji** [comment]
Your commit message should start with capital letters, 
https://www.google.jo/amp/s/amp.reddit.com/r/git/comments/1rnk90/should_i_capitalize_the_first_letter_of_my_commit/

File: database-mongo/index.js [hunk 1]
```diff
@@ -16,20 +16,22 @@ var doctorsSchema = mongoose.Schema({
...
 var Doctor = mongoose.model('Doctor', doctorsSchema);
 
 
 
-var userSchema = mongoose.Schema({
-  name: String,
+var adminSchema = mongoose.Schema({
+  username: {
+	type:String,
+	unique:true},
```
Comment (JackMiszencin) [inline database-mongo/index.js]: Put } on new line, at same indentation level as object's beginning
```
username: {
    type: String,
    unique: true,
}
```

File: server/index.js [hunk 2]
```diff
@@ -1,56 +1,153 @@
...
+  // check if the user exists and correct password
+  // render the doctorform template
+  var username=req.body.username;
+  var password=req.body.password;
+  //console.log(username);
+  dataModels.User.findOne({username:username},function(err,user){
+    if(err)   console.log("the user is not exist");
+      //res.redirect('/admin/login');
+  
+
```
Comment (JackMiszencin) [inline server/index.js]: Lots of unnecessary space.

File: server/index.js [hunk 3]
```diff
@@ -1,56 +1,153 @@
...
+app.post('/admin/signup',function(req,res){
+  //res.render('signup');
+   console.log(req.body);
+   // save to the database
+  var newUser=new dataModels.User(req.body);
+   newUser.save(function(err,doc){
+    if(err){
+      console.log("error in saving a new doctor");
+
```
Comment (JackMiszencin) [inline server/index.js]: Eventually, you'll want to be sending back an error response to the client, telling it that the request failed.

Comment (JackMiszencin) [reply server/index.js]: Same goes for some of the other if statements here. You'll need responses instead of console.log statements.

**Pull request merged**